src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -401,26 +493,69 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
+                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative-test/cml-example-cloud-gpu/pull/1">
                         <Image src="/img/github/cml-runner-report.png" alt="GitHub Cloud report example" />
                       </a>
                     </ExampleBox>
-
+        
+                  </Collapser>
+                )}
+                bitbucket={(
+                  <Collapser>
+                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-test/cml-example-cloud">
+                      <div><span>pipelines:</span></div>
+                      <div><span>  default:</span></div>
+                      <div><span>  - step:</span></div>
+                      <div><span>      name: deploy-runner</span></div>
+                      <div><span>      image: iterativeai/cml:0-dvc2-base1</span></div>
+                      <div><span>      script:</span></div>
+                      <div><span>        - |</span></div>
+                      <Tooltip type="runner">
+                        <div><span>          cml runner \</span></div>
+                        <div><span>              --cloud=aws \</span></div>
+                        <div><span>              --cloud-region=us-west \</span></div>
+                        <div><span>              --cloud-type=m5.2xlarge \</span></div>
+                        <div><span>              --cloud-spot \</span></div>
+                        <div><span>              --labels=cml</span></div>
+                      </Tooltip>
+                      <div><span>  - step:</span></div>
+                      <div><span>      name: run</span></div>
+                      <Tooltip type="runner">
+                        <div><span>      runs-on: [self.hosted, cml]</span></div>
+                      </Tooltip>
+                      <div><span>      image: iterativeai/cml:0-dvc2-base1</span></div>
+                      <div><span>      script:</span></div>
+                      <Tooltip type="dependencies">
+                        <div><span>      - pip install -r requirements.txt</span></div>
+                        <div><span>      - python train.py  # generate plot.png</span></div>
+                      </Tooltip>
+                      <Tooltip type="reports">
+                        <div><span>      - echo &quot;## Report from your EC2 instance&quot; &gt;&gt; report.md</span></div>
+                        <div><span>      - cat metrics.txt &gt;&gt; report.md</span></div>
+                        <div><span>      - echo &#x27;![](./plot.png &quot;Confusion Matrix&quot;)&#x27; &gt;&gt; report.md</span></div>
+                        <div><span>      - cml comment create report.md</span></div>
+                      </Tooltip>
+                    </Code>
+                    <ExampleBox title="CML Report">
+                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-test/cml-example-cloud">
```

Comment:
- [ ] TODO: @DavidGOrtega create this PR

```suggestion
                      <a target="_blank" rel="noreferrer" href="https://bitbucket.org/iterative-ai/cml-cloud-case/pull-requests/1">
```

Reply:
Ah! So the test repo is only re-creating the repos and not the PRs, right?

Reply:
as per [this](https://github.com/iterative/cml-examples-dev/issues/9#issuecomment-1305358149), it is meant to also keep the PRs up-to-date.

Reply:
Was done by Oliwabot